httpclient/src/test/kotlin/tbdex/sdk/httpclient/E2ETest.kt

```diff
@@ -41,15 +42,16 @@ class E2ETest {
       keyManager,
       CreateDidDhtOptions(
         verificationMethods = listOf(
-          Pair(
+          Triple(
```

Comment:
wish i could pass variable names to these like `jwk = JWK.parse..., pubKey = ..., id = ...`